pkg/apis/starlingx/v1/system_types.go

```diff
@@ -265,6 +265,20 @@ type SystemSpec struct {
 	// +optional
 	Location *string `json:"location,omitempty"`
 
+	// Latitude is the latitude geolocation coordinate of the system's physical
+	// location.
+	// +kubebuilder:validation:Pattern=^[a-zA-Z0-9\-_\. ]+$
```

Comment:
There's no restrictions on what this string should be?

Reply:
Ok, there's a comment on the StarlingX review from Matt Peters that the format has been specifically left generic:
https://review.opendev.org/c/starlingx/config/+/776670/14#message-d0bb4b7fb9c9fa04ba09fc9b11364b6ff7c49666